cmd/abapEnvironmentAssemblePackages.go

```diff
@@ -0,0 +1,679 @@
+package cmd
```

Comment:
I think with 600+ loc it makes sense to extract parts of the code to a dedicated package.